test/src/test/java/org/corfudb/integration/LogReplicationIT.java

```diff
@@ -1190,7 +1192,7 @@ private LogReplicationFSM startLogEntrySync(Set<String> tablesToReplicate, Set<W
         blockUntilExpectedAckTs.acquire();
         expectedAckTimestamp = srcDataRuntime.getAddressSpaceView().getLogTail();
 
-        // Block until the snapshot sync completes == one ACK is received by the source manager, or an error occurs
+        // Block until the expected ACK Timestamp is reached
         System.out.println("\n****** Wait until the wait condition is met");
         if (waitConditions.contains(WAIT.ON_ERROR)) {
```

Comment:
Change the system.out to log.in.

Reply:
System outs for testing are all over the place so even changing one does not do much. If this is something that we want we can make this in a separate PR where we change all of them.